app/services/manage_braintree_donation.rb

```diff
@@ -26,32 +29,39 @@ def create
 
   def queue_message
     {
-        type: 'donation',
-        params: organize_params
+      type: 'donation',
+      params: organize_params
     }
   end
 
   def organize_params
     {
-        donationpage: {
-            name:             "#{page.slug}-donation",
-            payment_account:  'Default Import Stub'
-        },
-        order: {
-            amount:         transaction.amount.to_s,
-            card_num:       card_num,
-            card_code:      '007',
-            exp_date_month: expire_month,
-            exp_date_year:  expire_year,
-            currency:       transaction.currency_iso_code
-        },
-        user: {
-            email:    member.email,
-            country:  member.country
-        }
+      donationpage: {
+        name:             "#{page.slug}-donation",
+        payment_account:  'Default Import Stub'
```

Comment:
@NealJMD I've only just learnt that `payment_account` needs to be set according to payment type and currency. We must set it to a value that exists on AK. The following are permitted:

```
PayPal USD
PayPal GBP
PayPal CAD
PayPal EUR
PayPal AUD

Braintree USD
Braintree CAD
Braintree AUD
Braintree GBP
Braintree EUR
```

Do you want to implement it as part of this PR? Otherwise, I'm happy to tackle it on another PR (branching from here).